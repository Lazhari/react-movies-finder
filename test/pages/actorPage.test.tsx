import React from 'react'
import { render } from '../testUtils'
import Actor from '../../pages/actors/[id]'

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/actors/90633',
      pathname: '',
      query: '',
      asPath: '',
    }
  },
}))

describe('Actor page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Actor />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
