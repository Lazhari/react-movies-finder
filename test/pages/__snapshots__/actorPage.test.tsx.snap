// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actor page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-3"
        >
          <div
            class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              class="MuiCardMedia-root makeStyles-media-2"
              style="background-image: url(https://image.tmdb.org/t/p/w276_and_h350_faceundefined);"
            />
            <div
              class="MuiCardContent-root"
            >
              <h2
                class="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit MuiTypography-gutterBottom"
              />
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
              >
                Birth day: 
              </p>
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
              >
                Place of birth: 
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-9"
        >
          <h1
            class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
          >
            Known For
          </h1>
          <div
            class="MuiGrid-root makeStyles-root-4 MuiGrid-container MuiGrid-spacing-xs-3"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
